docs/content/docs/learn-flink/overview.md

```diff
@@ -94,7 +94,7 @@ Streams can transport data between two operators in a `one-to-one `(or forwardin
 
 ## Timely Stream Processing
 
-For most streaming applications it is very valuable to be able re-process historic data with the same code that is used to process live data – and to produce deterministic, consistent results, regardless.
+For most streaming applications it is very valuable to be able to re-process historic data with the same code that is used to process live data – and to produce deterministic, consistent results, regardless.
```

Comment:
looks good - can you do the chinese version -or raise a Jira to track the Chinese translation please.